### PR TITLE
[stable/heartbeat] add several config values

### PR DESCRIPTION
**Pull request by kimxogus**
Signed-off-by: Taehyun Kim <kgyoo8232@gmail.com>

#### What this PR does / why we need it:
- add `priorityClassName`, `tolerations`, `hostNetwork` config value
- add `POD_NAMESPACE`, `NODE_NAME` and `NODE_IP`
    - localhost monitor can be customized with `NODE_NAME` or `NODE_IP` like below and it's more intuitive in kibana uptime ui
```yaml
  heartbeat.monitors:
  - type: icmp
    schedule: '*/5 * * * * * *'
    hosts: ["${NODE_IP}"]
    ipv4: true
    timeout: 16s
    wait: 1s
```
image

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Review by cpanato — APPROVED** [review]

File: stable/heartbeat/values.yaml [hunk 1]
```diff
@@ -58,6 +60,15 @@ resources: {}
   #  cpu: 100m
   #  memory: 100Mi
 
+priorityClassName: ""
+
+nodeSelector: {}
+
+tolerations:
```
Comment (cpanato) [inline stable/heartbeat/values.yaml]: sorry, lets keep without any tolerations. only if there is a reason for that

**Review by cpanato — CHANGES_REQUESTED** [review]

File: stable/heartbeat/values.yaml (hunk 1 above)
Comment (kimxogus) [reply stable/heartbeat/values.yaml]: I left this for backward compatibility. I agree that no tolerations by default would be proper. I'll remove that

Comment (kimxogus) [reply stable/heartbeat/values.yaml]: removed

**Pull request merged**